screens/RegisterScreen.tsx

```diff
@@ -73,13 +81,13 @@ const RegisterScreen = ({ navigation }: Props) => {
               style={authStyles.button}
               icon="login"
               mode="contained"
-              onPress={() => submit()}
+              onPress={() => handleRegister()}
             >
-              Sign Up
+              {loading ? 'Loggin in' : 'Register'}
```

Comment:
To reflect the correct action/page
```suggestion
              {loading ? 'Registering...' : 'Register'}
```